janitor/functions/complete.py

```diff
@@ -142,10 +151,50 @@ def complete(
         6      2        2         b     0.0    99.0
         7      2        3         b     4.0     7.0
 
+        Expose missing values via the index:
+
+        >>> index = pd.date_range('1/1/2000', periods=4, freq='T')
+        >>> series = pd.Series([0.0, None, 2.0, 3.0], index=index)
+        >>> DT = pd.DataFrame({'s': series})
+        >>> DT.index.name = 'dates'
+        >>> DT
+                               s
+        dates
+        2000-01-01 00:00:00  0.0
+        2000-01-01 00:01:00  NaN
+        2000-01-01 00:02:00  2.0
+        2000-01-01 00:03:00  3.0
+
```

Comment:
Is this spacing between the two code-blocks intentional? Note that this would split the code blocks into two when rendered, usually only meant when you want to make 2 code example blocks distinct from each other, but it seems unnecessary here.

![image](https://user-images.githubusercontent.com/30731072/219829529-8da93cf8-d540-40d7-815f-4188cbec56b9.png)